etc/kayobe/ansible/monasca_ceph_monitoring.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Deploy Monasca-rsyslog on Ceph nodes
+  hosts: ceph
```

Comment:
Which ceph cluster is this going to use? If it is the softiron cluster, I think we will collide with the existing Monasca Agent.